source/core/GenericFunctions.cpp

```diff
@@ -722,7 +722,7 @@ static bool GetMinimumArrayDimensions(std::vector<TypedArrayCoreRef> arrays,
     IntIndex *array0Lengths = arrays[0]->DimensionLengths();
     for (int i = 0; i < array0Rank; i++) {
         IntIndex dimension0 = *(array0Lengths + i);
-        for (int j = 1; j < Int32(arrayCount); j++) {
+        for (int j = 1; j < size_t(arrayCount); j++) {
```

Comment:
Actually I was thinking something like this:
for (size_t j = 1; j<arrayCount; j++)